src/redux/categories/categories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Categories = () => (
+  <div className="cat-list">
+    <p>Under Construction</p>
+  </div>
+);
+
+export default Categories;
```

Comment:
- [ ] Kindly extract this component into the `src/components` directory. At this point, this redux file should be empty.
- [ ] Remember to configure your react routes after these refactors.